### ISSUE TITLE
Make graphs accessible

### ISSUE THREAD

**Issue by terotik** [kausaltech/kausal-watch-ui#25]
Screenreader only reads the numbers on graphs without context.
- Progress (make accessible)
- Timescale (make accessible)
- Emission Scope (make sure description is accessible)

WCAG 2.1  1.1.1. A
WCAG 2.1  1.3.1. A

**Comment by terotik** [comment]
Been trying to hide plotly.js charts from screen readers with `aria-hidden` -wrapper and provide aria-label with description of content. But VoiceOver still keeps picking up text elements inside plotly svg to read. Not sure how to hide svg contents without editing the plotly generated markup.

**Comment by terotik** [comment]
Plotly graphs now hidden and sr-only "graph not accessible" label added. We backlog accessible version of the data for now and note this in accessibility report.
